searx/templates/oscar/result_templates/key-value.html

```diff
@@ -6,7 +6,7 @@
         {% continue %}
     {% endif %}
     <tr>
-        <td><b>{{ key|upper }}</b>: {{ value }}</td>
+        <td><b>{{ key|upper }}</b>: {{ value|truncate }}</td>
```

Comment:
There is a similar file the simple theme: https://github.com/asciimoo/searx/blob/ccacd2f2c4cca2f813e4784a8eb601a796519f02/searx/templates/simple/result_templates/key-value.html#L7

Reply:
Should we add `truncate` to that template as well? Or why do you mention it?

Reply:
> Should we add truncate to that template as well ?

Yes I guess the same constraint will apply to the content length.